tests/jwt_auth/test_jwt_middleware.py

```diff
@@ -39,7 +39,7 @@ def wildcard_route():
     asgi=ECHO_APP, secret="SECRET", allow_unauthenticated=True
 )
 APP_EXCLUDED_PATHS = JWTMiddleware(
-    asgi=fastapi_app, secret="SECRET", excluded_paths=["/docs", "/*"]
+    asgi=fastapi_app, secret="SECRET", excluded_paths=["/docs", "*"]
```

Comment:
We should remove `"*"` as this lets everything through.

Reply:
That will decrease the coverage. If we remove that, we need to add another FastAPI application to test the route with wildcards like this
```python
fastapi_app_wildcard = FastAPI()

@fastapi_app_wildcard.get("/")
def wildcard_route():
    return "Wildcard route test"

APP_EXCLUDED_PATHS_WILDCARD = JWTMiddleware(
    asgi=fastapi_app_wildcard,
    secret="SECRET",
    excluded_paths=["*"],
)

def test_excluded_paths_wildcards(self):
    client = TestClient(APP_EXCLUDED_PATHS_WILDCARD)

    response = client.get("/")
    self.assertEqual(response.status_code, 200)
    self.assertIn(
        b"Wildcard route test",
        response.content,
    )
```
Is that OK?

Reply:
I was thinking something like this:

```python
fastapi_app_wildcard = FastAPI()

@fastapi_app_wildcard.get("/")
def home_root():
    return "Root"

@fastapi_app_wildcard.get("/path/a/")
def sub_root():
    return "Sub route"

APP_EXCLUDED_PATHS_WILDCARD = JWTMiddleware(
    asgi=fastapi_app_wildcard,
    secret="SECRET",
    excluded_paths=["/path/*"],
)

def test_excluded_paths_wildcards(self):
    client = TestClient(APP_EXCLUDED_PATHS_WILDCARD)

    response = client.get("/")
    # Requires a token
    self.assertEqual(response.status_code, 403)

    # Is an excluded path, so doesn't need a token
    response = client.get("/path/a/")
    self.assertEqual(response.status_code, 200)
```

Reply:
OK. I will add your tests with one change
```python
def test_excluded_paths_wildcards(self):
    client = TestClient(APP_EXCLUDED_PATHS_WILDCARD)

    with self.assertRaises(HTTPException):
        response = client.get("/")
        # Requires a token
        self.assertEqual(response.status_code, 403)

    # Is an excluded path, so doesn't need a token
    response = client.get("/path/a/")
    self.assertEqual(response.status_code, 200)

```